app/components/Home.tsx

```diff
@@ -0,0 +1,45 @@
+"use client";
+
+//LIBRARY IMPORTS
+import { Grid } from "@chakra-ui/react";
+import { useTina } from "tinacms/dist/react";
+import { PageQuery } from "@/tina/__generated__/types";
+
+//LOCAL IMPORTS
+import Hero from "./Hero";
+import CompanyValues from "./CompanyValues";
+
+export default function Home(props: {
+  data: PageQuery;
+  variables: {
+    relativePath: string;
+  };
+  query: string;
+}) {
+  // HOOKS
+  const { data } = useTina(props);
+
+  // console.log("data props:", data);
+  // console.log("Page blocks:", data.page.blocks);
+
+  return (
+    <Grid templateColumns={"1fr"}>
+      {data.page.blocks?.map((block, i) => {
+        switch (block?.__typename) {
+          case "PageBlocksHero": {
+            return <Hero key={i} {...block} />;
+          }
+          case "PageBlocksValues": {
+            return <CompanyValues key={i} {...block} />;
+          }
+          // case "PageBlocksLocation": {
```

Comment:
Are these future cases for this switch case statement? If so, delete now if not needed for this PR then bring them back later on an update. If there are only 2-3 cases, you could also consider refactoring it to an if block for clarity, but not necessary.

Reply:
There's a block for each section of the home page, so there are still 4 more: Location, Our Machines, Request a Quote, Join Our Team (6 total)